pulseguardian/model/base.py

```diff
@@ -22,4 +24,10 @@
 
 
 def init_db():
-    Base.metadata.create_all(bind=engine)
+    while True:
+        try:
+            Base.metadata.create_all(bind=engine)
+        except OperationalError:
+            time.sleep(5)
```

Comment:
What do you think about adding a `log.info()` here saying "waiting for DB" or "Retrying DB connection" or ...?